codex-skill/SKILL.md

```
@@ -0,0 +1,364 @@
+export SESSION_ID=$(uuidgen | tr '[:upper:]' '[:lower:]')
+```
+
+**Why required?** ba's ownership model prevents multi-agent conflicts by requiring explicit session identification. Without it:
+- `ba claim` will fail (cannot establish ownership)
+- `ba mine` will fail (cannot identify your issues)
+- Multi-agent coordination breaks down
+
+### Error Handling
+
+If SESSION_ID is not set, ba commands will fail with:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)